client/src/views/admin/AdminRegistration.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div>
+    <b-container>
+      <b-card>
+        <b-container>
+          <h1>
+            <b>ADMIN REGISTRATION </b>
+          </h1>
+          <div class="form mb-3">
+            <b-form-input
+              v-model="name"
+              :state="userState"
+              placeholder="Name"
+            ></b-form-input>
+          </div>
+          <div class="form mb-3">
+            <b-form-input
+              v-model="email"
+              :state="userState"
+              placeholder="Email"
+            ></b-form-input>
+          </div>
+          <div class="form mb-3">
+            <b-form-input
+              v-model="password"
+              :state="passwordState"
+              type="password"
+              placeholder="Password"
+            ></b-form-input>
+          </div>
+          <b-button variant="outline-primary" size="lg" @click="registerAdmin"
+            >Register</b-button
+          >
+        </b-container>
+      </b-card>
+    </b-container>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "AdminRegistration",
+  watch: {
+    name() {
+      if (this.name.length > 2) {
+        return (this.userState = true);
+      }
+      this.userState = false;
+    },
+    email() {
+      if (this.email.length > 2) {
+        return (this.userState = true);
+      }
+      this.userState = false;
+    },
+    password() {
+      if (this.password.length > 2) {
+        return (this.passwordState = true);
+      }
```

Comment:
please add proper validations, I think `joi` validation is 7 character password
also I feel min 4-5 character would be file instead of 7